### DISCUSSION TITLE
Add more solidity problems

### DISCUSSION THREAD

**Discussion by eshaan41** [codewars/codewars.com#3093] [Ideas]
I feel like there are very less solidity problems in compared to other languages. I feel codewars has done a great job giving attention to it but we need more and more of it. Thank you.

**Comment by Kacarott** [comment]
The content you see on codewars is almost entirely community created. If you want more kata in Solidity, then you may have to either create more yourself, or convince other people to be interested enough in solidity to want to create some :)
